tests/read/test_get_api_requests.py

```diff
@@ -48,5 +59,7 @@ def test_get_api_requests_for_one_user_end_datetime(db_session):
         session=db_session,
         email=user.email,
         end_datetime=dt.datetime.now(tz=dt.timezone.utc) - dt.timedelta(hours=1),
+        include_in_url=None,
+        exclude_in_url=None,
     )
     assert len(requests_sql) == 0
```

Comment:
perhaps leave this test how they are, and add a new test, where we can clear see that the `include_in_url` is working